tests/components/sound.test.js

```diff
@@ -75,4 +71,19 @@ suite('sound', function () {
       assert.ok(sound.pause.called);
     });
   });
+
+  suite('play', function () {
```

Comment:
maybe you can test that `this.attrName` is correctly being used with `setObject3D`